src/components/BlogPost/BlogPost.js

```diff
@@ -46,24 +100,51 @@ function BlogPost(props) {
           <Grid>{blogSideDate}</Grid>
           {blogSideTags}
         </Grid>
-        <Grid item xs={9}>
+        <Grid item lg={9} xs={12}>
           <Link
+            className={width === 'xs' ? classes.linkSmall : classes.link}
             to={{
               pathname: `/blogs/${id}`,
             }}
           >
-            <h1>{name}</h1>
+            <span>{name}</span>
           </Link>
+          {/* <div
+            className={props.width === 'xs' ? classes.red : classes.blue}
+          >{`Current width: ${props.width}`}</div> */}
           {image && <img className={classes.image} src={image} alt={alt} />}
-          <p className={classes.description}>{description}</p>
+          <p
```

Comment:
You can get rid of the error of "nesting <p>" by changing this to <div>. Use elements according to their needs. If you use <p> here, it will be <p><p>whatever</p></p> when description gets resolved which doesnt make sense.